worksheet_A/TerminalHacking.cpp

```diff
@@ -37,10 +85,49 @@ int main()
 	for each (std::string word in options)
 	{
 		std::cout << word << std::endl;
+		
 	}
 
-	// TODO: implement the rest of the game
+	// Main game loop
+	while (livesRemaining > 0)
+	{
+		std::cout << "Please input a guess using only capital letters." << std::endl;
+		std::cin >> playerGuess;
+		for each (std::string word in options)
+		{
+			if (playerGuess != word)
+			{
+				validWord = false;
+			}
+
+			else
+			{
+				validWord = true;
+			}
+		}
+		if (validWord == true)
+		{
+			if (playerGuess != secret)
+			{
+				livesRemaining -= 1;
+				std::cout << "Number of lives remaning " << livesRemaining << std::endl;
+				similarityScore = checkSimilarity(secret, playerGuess);
+				std::cout << "Similarity " << similarityScore << "/" << wordLength << std::endl;
+			}
+
+			else
+			{
+				std::cout << "You win the game" << std::endl << "Type Quit to exit" << std::endl;
+				std::cin >> playerGuess;
+				livesRemaining = 0;
```

Comment:
Needs a comment explaining that this ends the loop. Or better, just use the `break` keyword.